### PR TITLE
OCPQE-24174: move imbcloud/nutanix/vsphere upgrade ci jobs to correct arch ci yaml

### PR DESCRIPTION
**Pull request by jianlinliu**
1. Correct issues introduced in https://github.com/openshift/release/pull/53557, currently only aws/gcp/azure support multi-arch payload.
2. Rename `ibmcloud-ipi-proxy-private` and  `ibmcloud-ipi-proxy-private-fips-rt` ci jobs to `ibmcloud-ipi-proxy-private-rt` for consistency across versions.

### CODE REVIEW

**Comment by jianlinliu** [comment]
@JianLi-RH @shellyyang1989 pls help review.

**Comment by shellyyang1989** [comment]
/lgtm

Thanks Jianlin for the fix.

**Comment by jianlinliu** [comment]
cc @liangxia to review.

**Comment by jianlinliu** [comment]
Did a rebase to resolve conflicts, no other change. 

/pj-rehearse ack

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml [hunk 1]
```diff
@@ -426,13 +426,29 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-disconnected
     workflow: cucushift-installer-rehearse-ibmcloud-ipi-disconnected-private
+- as: ibmcloud-ipi-mini-perm-f28
+  cron: 16 15 12 * *
+  steps:
+    cluster_profile: ibmcloud-qe
+    test:
+    - chain: openshift-e2e-test-qe
```
Comment (JianLi-RH) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml:434]: openshift-e2e-test-qe or openshift-upgrade-qe-test?

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml]: Done

**Pull request merged**